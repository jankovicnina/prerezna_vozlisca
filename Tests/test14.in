483 1168
408 113 2.46
223 57 8.84
346 10 3.18
319 86 2.95
78 351 6.3
348 120 9.59
458 378 7.76
378 301 8.78
266 256 1.69
233 450 2.36
344 69 9.68
372 249 7.8
16 324 5.35
303 280 -1.58
332 349 5.93
391 90 8.31
21 166 1.33
232 33 3.66
231 44 4.68
117 351 4.05
237 302 0.33
190 286 3.0
447 474 4.79
233 54 -0.18
451 219 6.61
124 287 -0.76
359 372 3.94
465 218 7.48
393 135 5.7
252 222 6.28
304 281 4.35
282 430 9.72
81 320 -0.23
234 295 7.05
318 266 7.58
55 102 0.87
4 470 5.1
420 439 0.96
183 294 8.7
358 469 2.2
49 390 8.98
230 367 2.9
402 353 -0.81
86 78 1.01
275 204 -0.58
72 280 -1.06
5 232 4.0
153 381 2.5
330 107 -0.72
58 8 0.2
331 157 7.99
76 365 -0.16
151 181 2.35
425 306 1.67
395 447 4.63
198 326 6.5
222 52 5.17
56 354 6.69
53 478 -0.73
117 450 -0.7
355 49 7.29
326 103 -0.29
325 254 7.46
435 173 3.01
325 132 1.01
84 451 5.17
148 198 5.17
217 241 -1.17
387 217 0.65
318 461 9.93
125 237 6.66
283 245 3.02
115 181 5.3
3 158 8.0
141 424 9.15
175 356 2.17
316 137 1.69
371 442 2.76
61 101 3.68
41 428 1.87
23 403 2.85
408 186 5.43
265 275 6.62
158 149 -0.0
153 34 6.75
208 316 0.84
218 230 8.63
40 182 8.79
439 415 2.53
339 469 5.79
468 256 8.49
244 330 6.58
291 130 6.56
100 70 8.21
272 323 4.59
311 90 8.74
167 339 2.07
86 87 9.46
436 408 7.46
449 455 -1.18
364 84 -1.46
136 256 3.25
447 200 3.27
412 322 8.17
136 470 7.48
337 434 4.15
124 1 5.61
82 172 6.8
377 111 2.16
33 261 -0.43
384 73 6.89
83 110 9.85
241 168 9.82
12 220 0.19
12 39 0.53
444 127 7.2
325 99 -1.07
121 235 9.23
292 347 7.76
157 49 9.29
203 71 4.93
468 35 2.56
96 407 8.93
288 114 7.07
350 60 7.36
245 26 5.52
307 237 -0.78
450 448 4.99
98 6 2.88
26 156 6.53
379 68 2.99
381 290 0.8
422 223 4.66
16 64 1.86
465 194 7.64
132 409 8.61
342 354 8.35
42 240 1.16
415 388 3.33
266 163 3.42
65 100 2.59
185 154 9.05
457 350 -1.23
334 419 5.96
215 423 3.94
394 191 0.68
382 137 3.03
409 432 -0.56
475 358 1.79
90 355 1.58
350 226 6.3
79 161 6.97
164 297 1.6
45 215 8.24
141 209 0.02
127 242 9.32
69 268 0.6
417 292 8.9
305 167 -1.14
450 171 9.9
322 43 7.1
404 85 9.26
277 206 2.18
398 125 -0.78
325 279 2.53
242 139 -0.88
39 295 7.19
284 246 9.19
454 170 0.01
445 247 1.16
127 265 6.83
433 16 6.03
301 185 5.17
38 14 7.48
27 97 0.42
442 51 8.58
421 246 8.36
377 94 6.28
467 145 -1.61
473 29 0.76
154 21 4.3
404 435 2.44
337 37 2.36
327 203 6.13
454 464 -0.69
140 288 -0.66
315 42 -1.83
267 433 7.4
116 302 3.64
367 217 4.21
139 44 0.85
391 434 3.94
10 201 5.37
37 285 5.5
90 366 8.78
178 346 0.95
11 20 4.91
104 396 4.95
242 270 3.38
118 398 9.11
134 23 -0.53
278 273 7.94
242 193 8.21
179 121 6.9
163 419 3.59
216 411 6.82
194 231 8.94
78 13 3.88
166 469 4.91
440 94 0.39
297 401 8.6
31 228 1.19
461 19 3.84
388 479 5.56
172 370 -1.05
232 231 4.38
239 44 3.46
427 481 4.4
292 124 5.05
310 70 4.22
276 55 -1.97
464 463 -0.03
184 446 1.89
439 346 1.03
32 239 9.68
112 40 2.97
288 423 3.42
399 184 0.81
403 176 3.84
50 406 8.72
480 7 8.58
371 274 5.9
446 437 4.29
296 63 7.79
243 53 -0.33
62 199 7.65
136 427 5.66
157 379 6.02
332 345 9.13
480 344 -1.03
211 292 7.18
60 356 6.69
374 357 6.0
97 405 2.5
207 454 6.09
87 207 2.46
25 226 4.51
372 413 7.36
206 338 8.02
60 0 2.23
370 471 8.94
385 423 6.01
56 313 6.22
387 278 8.2
160 119 4.84
261 255 0.03
339 89 3.14
164 229 9.05
50 343 -0.15
253 201 0.7
355 321 5.66
439 183 2.71
81 88 5.64
380 171 0.1
225 5 1.26
106 478 9.27
229 80 4.3
57 123 4.03
15 449 1.95
438 459 2.02
481 189 -0.88
266 357 8.77
85 402 -0.37
391 281 6.38
18 208 5.57
288 410 5.63
61 405 6.42
1 295 7.9
25 481 4.33
242 430 0.13
478 321 7.92
239 61 0.58
10 352 6.28
209 314 7.52
252 215 5.67
63 60 4.83
202 123 5.72
294 238 5.17
425 139 6.5
68 233 5.44
449 50 7.71
386 480 5.3
301 74 3.22
122 146 6.37
337 71 1.63
297 121 -0.32
11 317 9.03
57 457 0.85
477 415 9.6
462 275 9.51
230 163 0.01
304 55 3.38
46 190 0.75
458 195 7.62
246 178 4.18
199 442 4.09
442 141 -0.28
447 133 8.25
64 204 4.51
280 438 8.28
326 329 -0.38
196 387 7.87
161 352 4.92
89 85 7.4
284 471 1.41
221 59 4.26
33 326 3.28
358 277 -0.22
213 182 5.32
357 453 0.65
35 6 4.32
344 434 3.95
251 131 1.74
326 277 5.18
434 23 1.43
77 386 -1.11
124 276 6.38
75 174 -1.13
247 3 7.59
319 208 7.97
363 38 1.0
204 450 8.98
260 96 7.54
286 440 9.06
340 108 -1.74
55 69 2.31
339 38 5.97
482 219 0.62
68 98 8.69
445 372 9.84
137 289 8.03
123 278 8.47
88 454 0.27
349 58 6.95
272 147 4.13
474 69 6.37
325 264 2.48
273 199 0.37
28 224 4.02
143 359 2.14
53 39 6.16
19 84 7.3
437 115 0.17
271 276 3.17
153 349 -0.92
51 194 8.66
405 15 -1.63
83 366 9.69
277 305 3.83
198 27 1.19
409 54 9.67
116 405 1.24
70 122 5.78
1 375 2.97
93 428 8.84
95 119 1.79
199 436 7.28
148 419 -0.07
442 374 3.58
354 153 1.25
250 159 7.39
360 375 4.44
105 288 8.81
241 160 2.4
235 257 0.9
270 336 9.09
287 234 1.63
195 203 -0.06
112 83 -1.52
448 217 2.18
7 399 5.22
420 472 3.25
4 407 9.79
455 343 6.32
240 193 7.83
110 291 6.84
469 104 1.84
231 349 1.8
451 447 -1.11
476 112 8.89
132 41 7.3
141 32 0.41
443 232 5.92
401 378 1.03
117 276 5.43
120 262 3.97
41 470 0.75
422 298 5.38
412 431 5.37
205 354 1.19
130 227 5.82
399 470 1.82
321 426 -0.98
292 365 -1.61
249 212 3.39
403 175 3.82
24 196 6.11
281 90 -0.35
278 0 8.28
43 48 8.84
298 139 6.42
446 1 -0.23
444 144 1.52
383 287 7.01
161 132 5.18
421 304 9.63
114 360 -0.23
269 276 9.55
453 197 1.19
318 319 1.13
39 133 4.64
113 143 -0.21
77 439 4.97
92 136 2.13
177 189 1.78
376 326 3.02
161 116 3.1
469 23 -0.95
463 303 4.56
259 189 2.42
117 152 4.03
61 162 -1.29
449 467 -1.27
341 258 5.51
312 55 1.51
378 369 1.68
323 315 7.17
295 118 0.45
411 444 8.08
142 417 3.11
98 240 4.51
99 475 9.86
368 321 2.82
175 380 3.49
136 362 2.52
45 350 -0.76
328 199 -1.27
114 386 3.58
294 202 1.22
49 355 0.27
154 82 5.61
320 369 9.55
456 183 8.71
12 25 4.21
151 143 3.31
81 57 3.82
53 214 0.49
419 123 0.98
417 92 5.06
288 437 2.83
114 394 5.07
370 385 -0.87
461 194 5.29
271 215 3.03
139 172 8.65
223 115 2.87
58 283 0.31
66 64 0.57
300 105 3.44
309 48 9.12
83 451 8.7
4 446 3.27
471 296 6.71
122 437 7.27
191 9 1.62
378 112 6.58
34 341 6.84
184 110 -1.45
105 179 9.8
192 124 5.29
202 266 -0.68
345 185 8.57
310 256 -0.76
317 384 7.32
399 95 2.83
429 478 9.8
0 117 1.62
467 434 8.76
450 214 8.49
159 97 7.29
455 266 6.84
308 263 6.32
168 328 4.2
273 376 -1.38
279 136 9.77
102 424 3.92
254 140 9.57
423 146 2.25
448 443 4.03
209 263 6.84
464 304 7.49
263 267 8.81
160 128 1.71
411 265 9.62
258 263 2.98
36 481 4.58
64 136 1.05
236 136 -0.92
391 429 2.22
109 16 2.16
107 149 7.19
9 335 2.58
479 148 -1.51
186 31 6.35
360 106 3.81
123 346 5.83
477 242 8.44
15 251 8.19
331 229 -1.3
445 201 7.36
166 356 9.73
439 125 0.41
343 377 6.12
80 456 5.13
31 403 1.72
237 392 -0.11
392 275 4.95
278 187 -1.81
477 332 3.09
269 420 2.97
255 163 8.97
90 93 6.25
30 142 -1.39
430 293 8.08
173 469 0.95
237 373 7.79
65 31 8.56
156 251 9.58
147 0 7.6
473 226 4.73
166 285 2.06
392 238 6.01
126 232 3.83
209 420 7.17
299 175 -0.46
372 114 0.04
432 46 1.92
270 18 4.64
460 90 0.23
156 138 6.84
358 396 4.75
97 379 4.5
99 364 4.46
404 299 -0.44
269 31 -0.7
36 444 6.77
212 458 -1.25
166 31 4.76
20 107 3.81
32 309 -0.5
8 300 5.37
357 397 5.43
76 447 1.13
458 176 1.88
321 229 8.17
93 296 0.42
287 30 -0.98
462 218 4.23
324 192 -1.34
149 416 6.19
67 361 2.24
155 24 -0.06
169 240 7.58
255 278 9.0
25 285 2.81
237 271 8.58
201 258 3.34
233 248 5.35
352 100 5.57
361 431 -1.16
238 34 4.48
410 2 9.5
438 387 2.16
182 367 0.94
312 211 9.21
13 355 7.41
445 284 1.29
89 422 -0.92
257 462 6.19
55 292 -0.02
428 77 5.55
9 59 -0.82
144 475 1.76
362 347 2.34
72 469 7.54
431 32 6.4
314 400 1.31
246 454 2.91
375 342 3.41
22 271 -1.07
329 246 0.18
11 386 -0.54
235 97 -0.02
398 239 2.59
363 454 1.95
424 344 9.47
482 295 6.29
3 221 2.18
360 85 5.18
333 164 1.65
469 238 8.38
322 251 -0.48
248 190 -1.0
84 197 7.79
401 289 1.51
197 77 4.26
13 82 4.65
377 3 8.87
403 75 3.23
101 143 3.81
28 169 4.94
310 437 3.42
17 129 2.22
95 244 3.55
77 206 3.36
50 227 4.88
342 43 5.7
101 301 0.64
319 227 2.58
93 381 3.98
401 441 3.67
103 420 3.25
225 18 0.81
459 135 3.31
166 399 9.6
175 35 3.75
369 434 0.77
199 167 6.95
459 67 9.49
273 180 8.94
364 433 0.52
63 267 3.05
203 396 1.22
281 302 4.17
423 218 1.98
104 472 0.17
351 373 6.55
350 221 2.38
238 181 8.77
421 452 5.94
292 193 4.03
393 217 9.57
226 336 9.03
474 457 -1.21
277 471 -0.85
181 35 -0.56
136 458 7.18
355 86 2.76
252 56 6.86
114 71 6.98
389 416 2.19
249 169 3.23
236 25 3.97
293 123 -1.18
40 262 9.77
138 271 5.4
184 480 7.76
123 155 1.01
137 327 -1.98
75 325 1.2
217 289 8.3
395 433 9.17
189 177 2.86
327 71 7.88
8 126 7.11
442 63 3.2
352 144 7.96
256 59 4.83
385 172 3.31
181 127 -0.91
399 67 3.06
269 114 3.77
240 272 5.8
481 140 5.79
104 392 -0.76
327 134 -0.15
334 454 9.7
385 138 -0.42
288 286 5.7
243 362 0.87
125 447 0.33
284 256 6.3
91 202 5.26
80 439 0.8
268 198 1.04
248 382 0.15
128 343 0.91
54 162 1.64
188 314 2.09
208 378 6.4
21 405 0.78
47 269 -2.0
414 107 7.21
255 427 8.49
455 299 8.82
397 318 2.88
405 56 0.48
227 66 9.05
161 318 0.21
193 421 8.82
100 264 7.23
470 418 9.88
396 33 -0.35
277 135 6.29
237 51 8.33
398 188 8.25
461 416 -1.07
439 158 9.18
18 351 6.26
460 261 9.25
242 54 4.76
428 256 8.98
251 174 5.88
26 90 6.42
317 201 8.81
448 45 7.42
47 130 5.46
104 328 3.01
190 372 7.0
369 104 0.99
234 110 7.54
6 165 7.39
337 401 9.04
419 385 1.81
191 99 6.18
462 306 5.88
40 239 1.71
418 442 6.63
144 315 9.33
176 463 9.52
421 222 7.76
7 217 0.54
153 365 4.25
260 205 9.6
462 284 1.56
16 224 0.45
279 225 3.98
198 206 8.65
426 467 0.39
94 181 0.9
128 429 2.72
293 419 5.94
174 53 6.36
119 51 8.45
308 186 8.04
89 117 -0.38
224 93 -1.3
474 141 6.31
25 228 -0.36
448 95 9.97
189 70 5.24
152 361 9.67
472 371 5.82
251 307 1.4
273 57 -0.89
359 332 4.11
465 366 8.02
227 91 9.74
468 331 -0.09
209 78 3.22
219 212 9.42
215 229 2.67
229 250 0.09
18 125 3.34
239 207 9.83
103 109 -0.41
254 459 2.2
230 308 4.5
187 383 7.38
451 88 0.7
268 274 4.3
408 99 2.52
44 342 1.4
54 188 5.4
232 151 6.13
464 286 6.34
320 409 3.95
23 462 -1.94
387 400 3.3
132 236 2.03
174 441 7.36
454 253 0.78
317 124 9.41
160 192 0.91
461 337 -0.19
48 324 0.22
316 213 -1.04
426 1 8.87
353 445 2.76
241 249 0.51
296 204 8.01
354 82 5.06
126 20 2.65
96 391 0.91
481 254 4.76
198 66 3.58
406 323 4.97
225 383 7.19
302 473 8.07
385 65 2.74
31 27 5.28
412 190 9.29
405 145 8.97
152 352 8.91
216 260 6.16
240 231 1.32
57 460 7.82
323 71 9.29
142 308 7.82
90 313 9.37
285 188 0.59
0 289 9.04
35 418 9.47
221 155 7.63
162 175 6.88
156 333 -0.36
86 400 0.3
327 164 -0.95
42 383 7.52
5 156 7.56
352 19 -1.1
384 409 9.26
277 7 2.19
150 195 5.16
480 292 -1.24
138 425 7.9
104 209 4.04
117 303 9.38
133 247 3.34
40 340 8.97
380 14 7.02
466 60 1.01
324 244 3.26
256 207 -1.17
354 152 -0.31
292 480 3.44
140 260 9.24
461 397 2.18
202 263 0.12
91 59 9.71
158 82 1.33
333 95 -0.49
356 99 2.65
390 40 -1.17
414 337 9.51
418 482 9.75
188 17 3.68
307 76 6.75
330 162 3.53
257 318 7.18
409 397 9.99
166 476 1.75
125 320 6.6
408 300 9.94
452 343 4.46
409 263 3.22
186 218 6.41
381 170 7.58
341 474 8.23
256 273 4.31
282 208 0.9
463 465 4.74
97 383 3.76
238 270 5.34
83 250 7.26
299 95 0.64
111 38 7.93
355 262 3.67
9 334 1.66
269 381 9.4
121 344 8.75
207 185 2.86
64 414 2.1
214 377 7.36
52 285 4.47
460 116 3.21
417 210 6.77
20 81 7.66
5 348 6.52
165 431 6.66
7 63 0.71
146 233 -0.82
80 98 2.72
269 21 8.04
343 409 1.89
311 412 2.05
296 468 4.13
246 137 -0.13
2 68 5.11
0 177 6.92
60 51 3.8
335 128 0.03
441 67 -1.43
298 380 -1.97
295 80 -0.61
37 352 1.12
290 308 -0.74
413 47 8.53
309 447 6.7
186 440 -1.28
397 205 8.32
6 25 6.88
178 160 6.79
25 472 7.63
79 384 0.36
137 6 8.55
361 317 5.46
285 199 9.8
454 122 -0.47
287 156 3.87
230 51 1.17
299 164 9.52
148 348 4.44
193 346 0.88
67 379 4.67
124 435 5.16
101 269 7.25
357 299 8.0
289 296 3.13
399 155 -0.21
275 387 3.25
115 114 4.58
245 79 -0.39
171 229 8.23
219 393 -1.23
44 283 5.54
39 269 7.35
160 479 2.59
134 47 8.64
255 245 7.91
425 116 7.43
380 45 9.82
99 182 1.17
55 319 3.06
301 12 6.0
359 29 7.43
482 78 1.01
279 479 4.54
316 318 1.87
143 287 3.57
370 404 4.31
330 460 -0.6
149 91 -0.63
471 201 5.49
96 271 1.62
331 159 7.49
351 47 1.45
366 99 1.38
427 482 5.82
132 44 3.01
71 253 7.88
336 410 -1.28
282 33 4.74
263 153 9.24
193 404 5.39
463 228 3.72
373 311 2.72
186 382 5.2
422 320 5.84
416 117 4.49
268 158 -0.39
157 473 7.57
289 470 0.48
228 261 -1.18
412 57 5.38
200 305 8.83
157 338 2.53
111 406 8.82
259 412 -1.3
313 395 1.27
186 445 5.77
1 432 1.4
108 22 9.91
104 252 6.77
29 4 2.22
127 250 4.21
34 57 2.91
407 389 2.62
309 118 0.01
106 363 1.48
458 41 -1.01
305 54 8.48
159 312 6.4
111 445 3.52
120 34 5.24
342 32 5.59
458 267 3.3
40 65 7.2
120 65 0.05
365 314 3.6
478 158 4.71
164 219 3.71
70 323 3.33
429 241 4.96
237 414 4.77
98 58 8.47
30 220 1.14
94 265 6.78
262 139 7.12
210 272 6.74
82 138 2.0
209 70 8.98
78 56 8.57
332 202 8.82
466 429 -0.21
250 298 3.35
297 482 5.97
78 240 8.28
220 72 5.49
250 36 6.6
220 388 4.37
383 138 3.84
267 167 1.66
347 103 6.05
393 252 6.73
441 184 8.91
478 226 5.64
264 315 0.45
314 78 -0.64
330 93 -1.02
387 85 9.24
117 84 9.03
335 476 -0.01
443 366 2.05
148 288 -1.75
157 102 -1.1
366 249 5.19
441 45 7.78
373 334 8.88
170 180 7.32
145 105 8.93
97 74 -0.06
118 263 9.11
412 341 -0.32
272 194 4.5
138 269 0.15
134 466 6.33
452 403 1.57
31 176 9.62
91 277 2.19
171 175 5.28
327 228 0.63
365 290 1.9
317 442 8.06
296 100 1.96
94 391 0.27
41 402 8.4
339 42 9.2
228 49 3.86
325 158 6.56
114 77 8.4
207 351 -1.07
201 389 2.53
96 191 6.98
424 478 2.67
460 28 2.52
151 235 6.35
450 221 6.79
19 243 7.94
346 141 9.92
28 104 5.53
329 377 -0.9
62 437 5.24
189 274 -1.42
180 11 1.84
51 349 3.36
72 452 7.21
0 43 4.36
470 441 9.82
71 221 -0.7
311 216 3.63
389 477 3.78
169 244 0.2
458 400 -0.75
249 96 1.9
232 401 6.42
155 438 6.36
316 404 1.94
109 408 3.62
235 87 8.91
353 93 3.68
301 140 2.68
274 202 -0.92
402 299 2.85
37 360 6.7
264 136 8.2
299 403 7.61
210 126 6.97
60 334 6.92
14 282 3.06
249 280 5.07
202 405 1.17
148 336 3.33
59 223 3.95
55 111 8.64
348 298 5.9
387 305 6.4
395 272 4.91
400 259 9.94
131 41 8.86
381 254 6.32
419 464 1.8
141 331 8.16
91 411 1.32
353 39 -0.48
338 53 6.94
393 477 8.42
397 62 9.17
406 233 5.61
42 45 -1.28
448 21 0.71
249 118 8.88
374 214 7.27
440 159 2.6
193 206 7.16
285 209 1.0
140 452 5.71
74 461 5.31
294 270 2.6
135 131 8.31
290 50 7.13
307 436 5.22
160 13 7.62
2 231 1.31
479 192 4.16
30 451 6.78
284 466 9.18
300 325 8.15
480 79 -1.87
128 150 1.93
129 432 0.09
175 339 7.07
266 310 9.57
475 465 1.93
83 62 -1.81
146 25 4.02
433 30 8.71
405 175 1.36
293 421 0.45
316 188 5.03
434 254 -0.48
454 82 3.42
219 37 -0.31
73 66 0.59
214 399 4.06
423 111 -0.2
306 368 0.99
299 19 6.81
8 96 6.2
313 195 0.89
379 95 9.63
41 346 6.34
324 104 -1.27
278 463 4.9
413 92 5.96
163 374 -0.29
209 327 8.23
369 348 0.24
