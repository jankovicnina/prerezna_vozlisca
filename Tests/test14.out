796 7846 12666 15902 25638 44709 46275 47166 48474
